src/main/java/com/github/kay9/dragonmounts/dragon/TameableDragon.java

```diff
@@ -277,6 +284,12 @@ public void setFlying(boolean flying)
         entityData.set(DATA_FLYING, flying);
     }
 
+    public void setNavigation(boolean flying) {
```

Comment:
If it isn't too much trouble, I'd like to ask that this pr follows the code format for the rest of the project.
In this case, brackets are on next-lines.